pages/index.js

```diff
@@ -66,15 +66,31 @@ export default function Home({ isConnected }) {
     }))
   }
 
+  const updateCompletedHabits = (id, isCompleted) => {
+    setHabits(habits => habits.filter(habit => {
+      if(habit.id === id) {
+        habit.completed = isCompleted
+        return habit;
+      } else {
+        return habit
+      }
+    }))
+  }
+
+  useEffect(() => {
+    let count = 0;
+    habits.forEach(habit => {
+      if (habit.completed === true) count++;
+    })
+    setCompletedHabits(count);
+  }, [habits])
```

Comment:
Do you need a variable to keep track of a completed habits/have this useEffect at all? It seems like you could infer this by just looking at your `habits` array and using something like [filter()](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/filter) to get the # of completed habits.